core/src/string.rs

```diff
@@ -47,6 +47,13 @@ impl<'gc> AvmString<'gc> {
         Ok(Self::new_utf8(gc_context, utf8))
     }
 
+    pub fn new_utf8_bytes_lossy<'b, B: Into<Cow<'b, [u8]>>>(
+        gc_context: MutationContext<'gc, '_>,
+        bytes: B,
+    ) -> Self {
```

Comment:
Given that `String::from_utf8_lossy` always makes a copy, this should probably take a `&[u8]` instead of a `Cow<'_, [u8]>`.

Reply:
String::from_utf8_lossy does not always make a copy - it returns a Cow<'_, str>